### ISSUE TITLE
Export to Web doesn't work

### ISSUE THREAD

**Issue by Sauermann** [2shady4u/godot-sqlite#196]
**Environment:**
 - Linux -> Export to Web
 - Godot v4.3.stable.official [77dcf97d8]
 - Godot-SQLite 4.4 from the asset store

**Issue description:**
Exporting to Web causes the following error to show up in the browser console:
```
Uncaught (in promise) LinkError: imported function 'env._ZNSt3__29to_stringEx' signature mismatch
```
Also the program doesn't run.

Also note, that I was able to get it running in a browser, when I compile Godot export templates from master branch and SQLite with the command:
```
scons use_llvm=yes linker=mold target=template_debug platform=web dev_mode=yes dev_build=yes dlink_enabled=yes threads=no
```

I would expect, that there is a way to run a Godot-SQLite project in browsers using the published versions of Godot and Godot-SQLite.

**Steps to reproduce:**
1. Load MRP
2. Install official export templates and enable Web export
3. Install Godot-SQLite 4.4 from the asset store and activate it
4. run the project on a Web Browser

**Minimal reproduction project:**
[BugWebSQLiteSave.zip](https://github.com/user-attachments/files/18265597/BugWebSQLiteSave.zip)

**Additional context**
In the export options I have tried both settings: "Thread support On" and "Thread support Off"

![image](https://github.com/user-attachments/assets/34ce24e1-2608-4a8f-9d9e-e9001111fde2)

Related:
- https://github.com/godotengine/godot/issues/97426
- https://github.com/godotengine/godot/issues/94537

**Comment by JonnyTech** [comment]
https://github.com/2shady4u/godot-sqlite#supported-operating-systems

> Supported operating systems:
> HTML5 (stability depends on browser)

Which browser are you using?

**Comment by Sauermann** [comment]
Browsers: tested on Firefox and Chromium.

Just to make it clear: when I compile Godot, the Godot export templates and Godot-SQLite myself, the project works for me in Firefox.

**Comment by 2shady4u** [comment]
> Browsers: tested on Firefox and Chromium.
> 
> Just to make it clear: when I compile Godot, the Godot export templates and Godot-SQLite myself, the project works for me in Firefox.

This is a known issue unfortunately 😢 When Godot 4.4 gets released the godot-cpp repository should normally receive the latest commits which have proper support for Web export.

**Comment by Sauermann** [comment]
Thank you for letting me know, that this will be resolved in 4.4.

**Comment by upta** [comment]
> > Browsers: tested on Firefox and Chromium.
> > Just to make it clear: when I compile Godot, the Godot export templates and Godot-SQLite myself, the project works for me in Firefox.
> 
> This is a known issue unfortunately 😢 When Godot 4.4 gets released the godot-cpp repository should normally receive the latest commits which have proper support for Web export.

I'm running into this same issue using Godot 4.4.1 and using Godot-SQLite 4.4

With 

![Image](https://github.com/user-attachments/assets/1e37b458-3955-4233-9412-6baa1cb27005)
Chrome:
LinkError: WebAssembly.instantiate(): Import #126 "env" "memory": mismatch in shared state of memory, declared = 1, imported = 0

Firefox:
LinkError: imported function 'env._ZNSt3__29to_stringEx' signature mismatch

With

![Image](https://github.com/user-attachments/assets/c66bef83-3026-426a-ac04-18126ad80520)
Chrome:
LinkError: WebAssembly.instantiate(): Import #76 "env" "_ZNSt3__29to_stringEx": imported function does not match the expected type

Firefox:
LinkError: imported function 'env._ZNSt3__29to_stringEx' signature mismatch
